test/e2e/feature/feature.go

```diff
@@ -331,6 +331,21 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	ValidatingAdmissionPolicy = framework.WithFeature(framework.ValidFeatures.Add("ValidatingAdmissionPolicy"))
 
+	// owning-sig: sig-storage
+	// kep: https://kep.k8s.io/3751
+	// test-infra jobs:
```

Comment:
waiting for @msau42 feedback.

Reply:
https://github.com/kubernetes/test-infra/pull/31717 was merged.

Reply:
This section looks great! Thanks a lot.